cmd/bnsapi/main.go

```diff
@@ -68,13 +68,17 @@ func run(conf Configuration) error {
 	rt := http.NewServeMux()
 	rt.Handle("/info", &handlers.InfoHandler{})
 	rt.Handle("/blocks/", &handlers.BlocksHandler{Bns: bnscli})
+	rt.Handle("/account/nonce/address/", &handlers.NonceAddressHandler{Bns: bnscli})
+	rt.Handle("/account/nonce/pubkey/", &handlers.NoncePubKeyHandler{Bns: bnscli})
 	rt.Handle("/username/owner/", &usernameHandlers.OwnerHandler{Bns: bnscli})
+	rt.Handle("/username/resolve/", &usernameHandlers.OwnerHandler{Bns: bnscli})
 	rt.Handle("/cash/balances", &handlers.CashBalanceHandler{Bns: bnscli})
```

Comment:
`/cash/balances` with an address of `04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17` on the swagger page seems to be broken.  When I click `Execute` the Curl that's emitted is `curl -X GET "http://localhost:8001/cash/balances" -H "accept: application/json"`, ie it doesn't contain the address.